src/shared/utils/challenge-listing/filter.js

```diff
@@ -136,6 +136,10 @@ function filterByText(challenge, state) {
 
 function filterByTrack(challenge, state) {
   if (!state.tracks) return true;
+  // include code challenges with 'data science' tag in datasci track
+  if (_.get(state.tracks, 'datasci')) {
+    return _.includes(challenge.technologies, 'Data Science');
+  }
```

Comment:
@shubhendusaurabh This is wrong! Have you tried it yourself? It excludes MM challenges from data science listing, and does not include design or development challenges into the listing for, say develop + datasci, or design + datasci